challenge.md

```diff
@@ -3,12 +3,28 @@
 1. Answer the following questions:
 
    - How would you select from JavaScript an element `p` that has the class `text` and also the class `important`?
+
+    **ANSWER: const element = document.querySelector('p.text.important');**
+
    - How would you select from JavaScript a `button` element with class `button` and that is disabled?
+
+    **ANSWER: const buttonElement = document.querySelector('button.button[disabled]');**
```

Comment:
`button` is redundant in this case.
```suggestion
document.querySelector(".button[disabled]");
```

This selector also accepts this syntax:
```suggestion
document.querySelector(".button:disabled");
```